toolkit/docs/quick_start/quickstart.md

```diff
@@ -0,0 +1,128 @@
+# Quick Start Guide
+  - [Install Prerequisites](#install-prerequisites)
+  - [Clone CBL-Mariner](#Clone-CBL-Mariner)
```

Comment:
>(#Clone-CBL-Mariner) [](start = 23, length = 20)

This one doesn't work. I think the whole thing must be small-caps.

Reply:
fixed.